src/main/java/com/internet/shop/controllers/orders/DeleteOrderController.java

```diff
@@ -0,0 +1,26 @@
+package com.internet.shop.controllers.orders;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.service.OrderService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/orders/delete")
+public class DeleteOrderController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private OrderService orderService =
+            (OrderService) injector.getInstance(OrderService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String orderId = req.getParameter("id");
+        Long id = Long.valueOf(orderId);
+        orderService.delete(id);
+        resp.sendRedirect(req.getContextPath() + "/orders/admin");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/orders");
```
we need to update this after we changed servlet mapping